src/backend/palrange.h

```diff
@@ -24,6 +24,8 @@ namespace snmalloc
 
     static constexpr bool Aligned = pal_supports<AlignedAllocation, PAL>;
 
+    static constexpr bool ConcurrencySafe = true;
```

Comment:
While true at the moment, do we want to promise that all PALs have concurrency-safe (de)alloc routines?

Reply:
Good question.  I could say false instead?  Or we could add a feature flag to each Pal that says if it is true?

Reply:
The Pals have always been assumed to be concurrency safe so far.